README.md

```diff
@@ -215,7 +215,7 @@ import { Entry, EntryFields } from 'contentful';
 export interface TypeArtistFields {
   name: Contentful.EntryFields.Symbol;
   profilePicture?: Contentful.Asset;
-  bio?: CFRichTextTypes.RichText;
+  bio?: EntryFields.RichText;
```

Comment:
Changes in README.md besides this one are done by prettier auto-formatting. Let me know if you prefer to leave those changes out.